### ISSUE TITLE
Unnecessary option name for track details

### ISSUE THREAD

**Issue by Cwpute** [funmusicplace/mirlo#1247]
![Image](https://github.com/user-attachments/assets/167c91b7-abd7-42cb-b255-63fde244c730)
Here is a pattern: on the left, the name of the option, and on the right, the option itself which essentially says the same thing with other words or a different syntax.
Maybe this is important for accessibility ?
If not, the title of the options on the left should be removed for clarity (you don't need to read the same information twice). If it is important for accessibility, then choose a single word or a couple of words to describe these options. For example, from top to bottom i'd choose: "Listed artists" (no change here), "Individual sale" and "Hype Circle promotion" (or simply "Promotion").

**Comment by timglorioso** [comment]
1. It's true that checkbox and radio button labels can be self-sufficient. In this case, Promotion benefits from some details.
2. For Individual Sale, removing the left part may disrupt the visual flow of information and make the setting easier to miss. Just speculation.
3. Redundant information isn't particularly helpful regardless of whether someone is using their eyes or a screen reader.
4. It looks like neither label is associated with the checkbox (`for` requires `id`, not `name`), so that's something to address.
5. I would consider adjusting the entire track editing layout so that labels are above their inputs, like in the small screen layout.

**Comment by timglorioso** [comment]
I've started working on this one. Converting the table to a list to improve semantic use of HTML.
